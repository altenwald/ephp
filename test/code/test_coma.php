<?php

$a = 10, $b = 100;
var_dump($a);
var_dump($b);
