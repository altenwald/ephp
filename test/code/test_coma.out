
Parse error: syntax error, unexpected ',' in {{CWD}}/test/code/test_coma.php on line 3
